### ISSUE TITLE
Figure formatting

### ISSUE THREAD

**Issue by agitter** [greenelab/meta-review#234]
Figure 1 was converted and needs to be reviewed.  Figure 2 is okay.  Figure 3 needs to be resized, and they have concerns about the resolution.  See https://github.com/greenelab/meta-review/issues/226#issuecomment-497856572

**Comment by dhimmel** [comment]
@agitter what changes were made to Figure 1 as per:

> Figure 1 has been modified to meet with formatting requirements. Please review the revised figure to ensure that it meets with your approval. Please note that if you approve of this revised figure, you do not need to upload it to the system.

**Comment by agitter** [comment]
Here are all the files attached to the submission currently
![image](https://user-images.githubusercontent.com/3532898/58748460-ecf79c80-843e-11e9-8f9f-c98a6cc03b99.png)

Here is the `Final Figure File` they created for `Fig 1`.  It's a zipped `.eps` because GitHub won't take `.eps` in attach file uploader:
[Fig1.zip](https://github.com/greenelab/meta-review/files/3243753/Fig1.zip)

Do you see any major differences?  The shade of purple may have changed.

**Comment by agitter** [comment]
@vincerubinetti do you have thoughts about what to do for Figure 3?  The key parts of the journal's comments are:
> Figure 3 exceeds the dimension limits for Figures. Figures must be less than 19.05cm in width, and less than 22.225cm in height. Please note that figures will be typeset at the dimensions provided. Please make the necessary adjustments and re-upload your Figures.
In Photoshop: Image→Image Size. Make sure the Resample Image checkbox is NOT checked. If the Width is over 19.05cm, type 19.05 in the Width box (19.05cm is our maximum allowable width for figures).
In GIMP: Image→Scale Image. Set the units of measurement, in the pull down menu next to Height, to millimeters. If the Width is over 190.5mm, type 190.5 in the Width box (19.05cm is our maximum allowable width for figures) and hit Tab. The new Height of the figure will appear, scaled proportionately to the change in Width.

> Figure 3 looks like it was originally at a lower resolution and will appear blurry or pixelated if published. Text and images should appear crisp and clear. Please note that this problem will not be detected or corrected by the PACE tool, and this will not be fixed by simply increasing the resolution of your Figures in the image editing software. Please carefully review the PLoS Figure Guidelines and provide Figure files that were originally created at 300 pixels/inch (dpi) and at least 6.68cm wide, but less than 19.05cm and under 22.225cm tall. For further information, please see the Figure Guidelines (http://journals.plos.org/ploscompbiol/s/figures).

The sizing shouldn't be an issue.  What about the resolution?

**Comment by dhimmel** [comment]
# Figure 3

Okay so Figure 3 is a TIFF, located [here](https://github.com/greenelab/meta-review/blob/362b839108fdcb109ede7dc29a2492303cec62fe/plos-comp-bio/figures/Fig3.tif).

> Figure 3 exceeds the dimension limits for Figures. Figures must be less than 19.05cm in width, and less than 22.225cm in height. Please note that figures will be typeset at the dimensions provided. 

Vince mentioned in the past that raster images don't have size dimensions, just a number of pixels. If this is true, I'm not sure how PLOS is determining that the dimension is too large (maybe by text size). I wonder if we can just specify that Figure 3 should be given width equal to 19.05cm (7.5 inches). Note the current pixel dimensions are 2176 (width) by 1888 (height).

At 7.5 inches wide, Figure 3 is 290 pixels per inch (2176 / 7.5), which is close to the requested 300 dpi.

**Comment by vincerubinetti** [comment]
I should clarify that image formats might support a dpi metadata, but in most cases (at least for me) the dpi is irrelevant, and a simple 1 to 1 pixel mapping is used (eg web browsers seem to ignore any dpi metadata). See this StackOverflow post for more info.

https://stackoverflow.com/questions/27323561/do-pngs-or-jpgs-have-a-dpi-or-is-it-irrelevant-when-building-for-retina

Tiffs, if I recall correctly, do support dpi and often have that dimension respected because I think tiffs are most often used for print?

**Comment by vincerubinetti** [comment]
@agitter Sorry for the delay in response.

Unfortunately I think I may have to go and recreate that whole figure at a higher resolution.

**Comment by dhimmel** [comment]
> I should clarify that image formats might support a dpi metadata

Okay that makes sense then why we would have failed that PLOS check. Let's have the new figure set a DPI that results in a Figure width of 7.5 inches. 600 dpi or greater at 7.5 inches would be ideal. The PLOS figure guidelines state:

> Resolution below 300 results in blurred, jagged or pixelated published figures. ​Resolution above 600 dpi may lead to resizing of the published figure.

So we should probably submit at 600 dpi. I imagine users may zoom in for this figure.

**Comment by dhimmel** [comment]
Note that to remake the Hypothesis panel of Figure 3, we will need to deal with the relocation of the TPOT repo and transfer of annotations as per https://github.com/greenelab/meta-review/pull/189#issuecomment-480323358.

**Comment by dhimmel** [comment]
# TPOT-FSS panel

I see that if we add the following to the [manuscript HTML](https://github.com/trang1618/tpot-fss-ms/blob/859af7b8e617098efe812e8d79a8d074f05967f9/manuscript.html) `<head>`, then the annotations transfer:

```html
  <link rel="canonical" href="https://trang1618.github.io/tpot-ds-ms/" />
```

Here's a version of the HTML with that line added for convenience: [tpot-fss-article.zip](https://github.com/greenelab/meta-review/files/3273672/tpot-fss-article.zip).

**Comment by vincerubinetti** [comment]
Addressed by #240
